App/src/test/java/vut/fit/gja2023/app/LoadingFromCsvTests.java

```diff
@@ -0,0 +1,120 @@
+package vut.fit.gja2023.app;
+
+import com.opencsv.exceptions.CsvException;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import vut.fit.gja2023.app.service.CsvReaderService;
+import vut.fit.gja2023.app.util.CsvReaderConfig;
+
+@SpringBootTest
+class LoadingFromCsvTests {
+    
+        private static final String TEST_FILE_ONE_PATH = "D:\\Projects\\GJA_2023\\App\\src\\test\\java\\vut\\fit\\gja2023\\app\\Resources\\test.csv";
+        private static final String TEST_FILE_TWO_PATH = "D:\\Projects\\GJA_2023\\App\\src\\test\\java\\vut\\fit\\gja2023\\app\\Resources\\test2.csv";
```

Comment:
Absolutne cesty nie, ono aj tak ked to spravis ako unit test tak si len nasimulujes ze to precita nieco z file systemu, ale pre buducnost ,keby chces nieco taketo spravit, tak dat to do resources ktore su jednu uroven pod main packagom, a odtial vyberat napr tymto: `URL url = this.getClass().getClassLoader().getResource(fileName);`

Reply:
Pro testování readeru jsem použil možnost s files v resources, protože jsem potřeboval, aby to opravdu přečetlo ten soubor a ne jen mock file.